src/main/java/com/github/retro_game/retro_game/config/SecurityConfig.java

```diff
@@ -38,6 +38,8 @@ protected void configure(HttpSecurity http) throws Exception {
             "/combat-report",
             "/espionage-report",
             "/join",
+            "/resetPassword",
+            "/changePassword",
```

Comment:
NIT: The project typically uses URLs `in-this-form`. To be consistent, it would be nice if those URLs are also in this form.

Reply:
Changed